tests/contrib/minikube/test_kubernetes_executor.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+import unittest
+from subprocess import check_call, check_output
+
+import requests
+import time
+import six
+
+try:
+    check_call(["kubectl", "get", "pods"])
+except Exception as e:
+    raise unittest.SkipTest(
+        "Kubernetes integration tests require a minikube cluster;"
+        "Skipping tests {}".format(e)
+    )
+
+
+class KubernetesExecutorTest(unittest.TestCase):
+
+    def test_integration_run_dag(self):
+        host_ip = check_output(['minikube', 'ip'])
```

Comment:
maybe this can be in a before_all step so it won't need to be repeated for future tests?